src/main/java/org/openelisglobal/common/controller/BaseController.java

```diff
@@ -28,7 +28,7 @@
 import org.springframework.web.servlet.support.RequestContextUtils;
 
 @Component
-public abstract class BaseController implements IActionConstants {
+public abstract class BaseController extends BaseRestController {
```

Comment:
Changing the Base Class will definately cause some things to break.
Can instead Leave the  getSysUserId method here